docs/code-style-guide.md

```diff
@@ -103,6 +114,51 @@ If you think some ASCII art will help explain something better, go for it! [asci
   float power(float a, float b);
   ```
 
+### Curly Braces
+
+* Opening and closing braces \(`{}`\) must be on the same indentation level.
+  ```cpp
+  // Incorrect
+  int foo() {
+      return 0;
+  }
+
+  // Correct
+  int foo()
+  {
+      return 0;
+  }
+  ```
+* Avoid ternary operators. Clarity is more important than line count.
```

Comment:
Not sure if ternary operators belong in a "curly braces" section?

Reply:
ok moved to misc